crates/bevy_text/src/text.rs

```diff
@@ -359,12 +381,12 @@ pub fn detect_text_needs_rerender<Root: Component>(
         (
             Or<(
                 Changed<Root>,
-                Changed<TextStyle>,
+                Changed<TextFont>,
                 Changed<TextLayout>,
                 Changed<Children>,
             )>,
```

Comment:
Removing `TextColor` here means animated text colors will not cause glyphs to be updated properly.

Reply:
> Removing `TextColor` here means animated text colors will not cause glyphs to be updated properly.

I don't follow? Animated colors shouldn't cause an update, isn't that the point?

Reply:
> I don't follow? Animated colors shouldn't cause an update, isn't that the point?

The glyphs need to update otherwise the color changes won't show up.

Reply:
Oh hold on, this is my mistake: color is always copied from entities when doing text extraction.